pkg/gateway/grpc.go

```diff
@@ -85,11 +85,10 @@ func (s *GatewayGRPCServer) ListenAndServe(ctx context.Context) error {
 		}),
 		grpc.ChainStreamInterceptor(otelgrpc.StreamServerInterceptor()),
 		grpc.ChainUnaryInterceptor(otelgrpc.UnaryServerInterceptor()),
-		grpc.MaxRecvMsgSize(32*1024*1024), // 32MB
```

Comment:
This will leave the max message size to [~2GB](https://github.com/grpc/grpc-go/blob/a1693ec5d2b165a14705859618e70895315967f9/server.go#L380-L386). Is this intended?

Reply:
The max send message size is in the other direction.